csharp-selenium-webdriver-sample/azure-pipelines.yml

```diff
@@ -29,52 +29,78 @@
 # an example of using the same test code with different combinations of browsers and OSes.
 #
 # For more information, see:
-# * https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started-multiplatform
-strategy:
-  matrix:
-    win-chrome:
-      imageName: windows-2022
-      browser: chrome
-    linux-firefox:
-      imageName: ubuntu-22.04
-      browser: firefox
+# * https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started-multiplatform# Copyright (c) Microsoft Corporation. All rights reserved.
+parameters:
+- name: strategy
+  type: object
+  default:
+  - name: win_chrome
+    imageName: windows-2022-secure
+    browser: chrome
+    os: windows
+  - name: linux_firefox
+    imageName: ubuntu-20.04
```

Comment:
Do we have the option of staying with `ubuntu-22.04`, instead of moving back to the older version?